lib/go/edgecontext/token.go

```diff
@@ -1,15 +1,15 @@
 package edgecontext
 
 import (
+	"github.com/golang-jwt/jwt/v4"
 	"github.com/reddit/baseplate.go/timebp"
-	jwt "github.com/reddit/jwt-go/v3"
 )
 
 // AuthenticationToken defines the json format of the authentication token.
 type AuthenticationToken struct {
 	jwt.StandardClaims
 
-	// NOTE: Subject field is in StandardClaims.
+	// NOTE: Subject field is in RegisteredClaims.
```

Comment:
It looks like you stopped using `RegisteredClaims` and went back to `StandardClaims`?

Reply:
😅